docker/everest-docker-image/Dockerfile

```diff
@@ -0,0 +1,110 @@
+# syntax=docker/dockerfile:1
+FROM debian:11.0 AS builder
+
+ARG REPO
+ARG BRANCH
+ARG EVEREST_CONFIG
+ARG OCPP_CONFIG
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+    git \
+    rsync \
+    wget \
+    cmake \
+    doxygen \
+    graphviz \
+    build-essential \
+    clang-format \
+    clang-tidy \
+    cppcheck \
+    libboost-all-dev \
+    maven \
+    openjdk-11-jdk \
+    nodejs \
+    npm \
+    libsqlite3-dev \
+    python3-pip \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /workspace/everest
+
+USER root
+
+# to avoid caching
+ARG BUILD_DATE=Unknown 
+
+# add github to known hosts
+RUN mkdir ~/.ssh
+RUN ssh-keyscan github.com > ~/.ssh/known_hosts
+
+RUN mkdir -p /workspace/everest/cpm_source_cache    
+ENV CPM_SOURCE_CACHE="/workspace/everest/cpm_source_cache"
+
+RUN git clone https://github.com/EVerest/everest-cmake.git
+RUN git clone https://github.com/EVerest/everest-utils.git
+WORKDIR /workspace/everest/everest-utils/ev-dev-tools
+RUN python3 -m pip install .
+WORKDIR /workspace/everest
+RUN git clone https://github.com/EVerest/everest-dev-environment.git
+WORKDIR /workspace/everest/everest-dev-environment/dependency_manager
+RUN python3 -m pip install .
+WORKDIR /workspace/everest
+RUN git clone https://github.com/EVerest/ext-switchev-iso15118.git
+WORKDIR /workspace/everest/ext-switchev-iso15118
+RUN pip install --user -r requirements.txt
```

Comment:
Why are the switchev dependencies installed here? Couldn't this be done in the second stage? Note that switchev uses peotry, which would be more concise to use instead of the requirements.txt.